docs/_articles/en/einstein/einstein-example-prompts.md

```diff
@@ -79,15 +79,19 @@ return orderItems;
 **Prompt:** Write an Apex trigger to add the number of contacts associated with an account.
 
 ```
-trigger gptcontacttrigger on Account (after insert, after update) {
-    List<Contact> listContacts = new List<Contact>();
-
-    for (Account acc : Trigger.new) {
-        List<Contact> listaccContacts = [SELECT Id, Name, AccountId FROM Contact WHERE AccountId = :acc.Id];
-        if (acc.Contacts.size() < listaccContacts.size()) {
-            acc.addError('Not enough contacts');
+// Write an Apex trigger to add the number of contacts associated with an account
+
+trigger AccountTrigger on Account (before insert, before update, before delete, after insert, after update, after delete, after undelete) {
+    
+    if (Trigger.isBefore) {
+        
+    } else if (Trigger.isAfter) {
+        if (Trigger.isUpdate) {
+            List<Account> accountsWithChangedContacts = [SELECT Id, NumberOfContacts FROM Account WHERE Id IN :Trigger.newMap.keySet()];
+            for (Account a : accountsWithChangedContacts) {
+                a.NumberOfContacts = Trigger.newMap.get(a.Id).NumberOfContacts;
+            }
```

Comment:
I ran this through the scanner and had a few flagged
`	"message": "Sev3: Validate CRUD permission before SOQL/DML operation or enforce user mode",`
was the only one that really concerned me.  I'm going to have Chris take a look but I think we should add to the prompt request that `WITH USER_MODE` be added to the SOQL query.

Reply:
Chris says he will review by the end of the day :+1:

Reply:
@capeterson is it OK for us to merge this PR? Thanks!